internal/command/arguments/apply.go

```diff
@@ -57,6 +57,14 @@ func ParseApply(args []string) (*Apply, tfdiags.Diagnostics) {
 		))
 	}
 
+	if apply.State.StatePath != "" {
+		diags = diags.Append(tfdiags.Sourceless(
+			tfdiags.Warning,
+			"state is deprecated",
+			fmt.Sprintf("use path variable of local backend instead : https://developer.hashicorp.com/terraform/language/settings/backends/local\n"),
```

Comment:
```suggestion
			"Deprecated flag: -state",
			fmt.Sprintf("Use `path` attribute within the `local` backend instead: https://developer.hashicorp.com/terraform/language/v1.10.x/settings/backends/local#path"),
```

Just a couple of nits over the language here. 

I'd want to call out that it's the flag specifically that is deprecated.

Also, the documentation can change over time so I think it's better for us to link to the v1.10.x documentation directly given we're adding this deprecation warning for the 1.10 release. That way if we change the `local` backend in the future, people using this version will get a direct link to something that makes sense.

Can you copy these changes over to the other places we raise the warning?

Thanks!

Reply:
Yeah, that makes sense. 
Thanks